### PR TITLE
Use nodemon to restart server and client while developing

### PR DESCRIPTION
**Pull request by geramirez**

### CODE REVIEW

**Comment by geramirez** [comment]
@kiranjthomas more changes to help make developing easier.

File: package.json [hunk 1]
```diff
@@ -4,10 +4,17 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npm run build && cd .. & node index.js",
-    "startProd": "node index.js",
+    "startDev": "nodemon",
+    "build_client": "cd client &&  elm make src/Main.elm --output=../public/index.js && cd ..",
+    "start": "node index.js",
```
Comment (kiranjthomas) [inline package.json:9]: what do you think about adding `--inspect` here so we can debug in the browser? https://nodejs.org/en/docs/guides/debugging-getting-started/

**Comment by kiranjthomas** [comment]
LGTM! I was able to get the project up and running. Thank you!

File: package.json (hunk 1 above)
Comment (geramirez) [reply package.json:9]: sure let's add it.

**Pull request merged**